client/stylesheets/_toCat.scss

```diff
@@ -5747,6 +5747,34 @@ textarea {
   font-size: 0.9rem !important;
 }
 
+.helpDiv h2, .helpDiv h3, .aboutDiv h2, .aboutDiv b, .faqdiv h2, .faqdiv h3, .safetydiv h2, .safetydiv h3, .reportdiv h2, 
+.reportdiv h3, .prohibiteddiv h2, .termsDiv h2, .contactdiv h2 {
+    font-weight: 400 !important;
+}
+
+.helpDiv, .aboutDiv, .faqdiv, .safetydiv, .reportdiv, .prohibiteddiv, .termsDiv, .contactdiv {
+    margin-bottom: 100px !important;
+}
+
+.helpDiv a, .aboutDiv a, .faqdiv a, .safetydiv a, .reportdiv a, .prohibiteddiv a, .termsDiv a {
```

Comment:
Selector `helpDiv` should be written in lowercase with hyphens<br>Selector `aboutDiv` should be written in lowercase with hyphens<br>Selector `termsDiv` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line